x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_select.js

```diff
@@ -15,11 +15,17 @@ import {
   EuiSelectable,
 } from '@elastic/eui';
 import { SymbolIcon } from '../legend/symbol_icon';
+import { SYMBOL_OPTIONS } from '../../symbol_utils';
+import { getUiSettings } from '../../../../../kibana_services';
 
 function isKeyboardEvent(event) {
   return typeof event === 'object' && 'keyCode' in event;
 }
 
+function getIsDarkMode() {
+  return getUiSettings().get('theme:darkMode', false);
+}
```

Comment:
Should we just put this function in `kibana_services`? Then you could also leverage it in `symbol_utils` > `getIconPaletteOptions()`. Alternatively, do you think it's worth having this function at all?

Reply:
ha crossposted with https://github.com/elastic/kibana/pull/71482#discussion_r454419919

Reply:
Ha, nice. Must've been the mind meld :vulcan_salute: